src/ansys/fluent/core/solver/flobject.py

```diff
@@ -410,6 +410,8 @@ def get_attr(
     def _is_deprecated(self) -> bool:
         """Whether the object is deprecated in a specific Fluent version.'"""
         deprecated_version = self.get_attrs(["deprecated-version"])
+        if deprecated_version:
+            deprecated_version = deprecated_version.get("attrs", deprecated_version)
```

Comment:
Extra unwrapping required in PyConsole. Without this `_is_deprecated` was returning `None` in PyConsole and that's why the old journal was passing in PyConsole (#3945).